src/Symfony/Component/JsonPath/Tokenizer/JsonPathTokenizer.php

```diff
@@ -26,9 +25,11 @@ final class JsonPathTokenizer
     private const BARE_LITERAL_REGEX = '(true|false|null|\d+(\.\d+)?([eE][+-]?\d+)?|\'[^\']*\'|"[^"]*")';
 
     /**
+     * @param callable(string $functionName, list<string> $args): void|null $functionValidator
```

Comment:
```suggestion
     * @param callable(string $functionName, list<string> $args): void $functionValidator
```

`void|null` does not make sense

Reply:
It's actually that the param is either a callable returning void, either null. I agree it looks ambiguous, but if I'm correct, CS fixer forces this order.